NodePhantomJS/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM selenium/base:2.44.0
```

Comment:
Is there a reason you chose to derive from base and not node-base? For example both the Chrome and Firefox nodes derive from node-base: https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeChrome/Dockerfile#L1